movie-browser/src/App.js

```diff
@@ -1,7 +1,36 @@
-import React from 'react';
+import React, { Component } from 'react';
+import MovieList from './components/MovieList';
+import axios from 'axios';
 
-const App = () => {
-  return <div>Movie Browser</div>;
-};
+const URL = 'https://ghibliapi.herokuapp.com/films';
+
+class App extends Component {
+  state = {
+    movies: [],
+  };
+
+  componentDidMount() {
+    axios
+      .get(URL)
+      .then(res => {
+        this.setState({
+          movies: res.data,
+        });
+      })
+      .catch(err => {
+        console.log(err);
+      });
+  }
+
+  render() {
+    const { movies } = this.state;
+    return (
```

Comment:
add a newline between the variable declaration and the return statement

Reply:
done